cmd/snapshotgc/snapshotgc.go

```diff
@@ -182,3 +249,31 @@ func deleteSnapshots(
 		}
 	}
 }
+
+func main() {
+	var prSnapshotsToKeep, nonPrSnapshotsToKeep int
+	flag.IntVar(
+		&prSnapshotsToKeep,
+		"pr-snapshots-to-keep",
+		5,
+		"Number of PR snapshots to keep after garbage collection",
+	)
+	flag.IntVar(
+		&nonPrSnapshotsToKeep,
+		"non-pr-snapshots-to-keep",
+		5,
+		"Number of non-PR snapshots to keep after garbage collection",
+	)
+	flag.Parse()
+
+	cl, _ := client.New(config.GetConfigOrDie(), client.Options{Scheme: scheme})
+	zc := zap.NewProductionConfig()
+	z, _ := zc.Build()
+	logger := zapr.NewLogger(z)
```

Comment:
@jsztuka @MartinBasti can I have your help with setting up the log options so it fits whatever we use in other places?

Reply:
https://github.com/redhat-appstudio/integration-service/blob/51999944dcf355549f6292df6649494a5747507f/main.go#L77 can we have something like this?

Reply:
done. It's writing logs to the terminal on my dev env.